args/misc.py

```diff
@@ -1,10 +1,15 @@
+from data.movement import MovementActions
+
+
 def name():
     return "Misc."
 
 def parse(parser):
     misc = parser.add_argument_group("Misc.")
-    misc.add_argument("-as", "--auto-sprint", action = "store_true",
-                      help = "Player always sprints. Sprint Shoes have no effect")
```

Comment:
Would it be possible to leave it in for now, but just have it call the other flag? Removing this option would break every saved seedbot flag which uses the -as flag. At some point (maybe when the major release happens?) we'd want to remove these deprecated flags, but for now cleaning up all of the existing flagsets would prove problematic.

Reply:
I could just do a find & replace on the preset file if we went this way - I don't think it would be a huge lift

Reply:
Added the argument back, and going to completely ignore it..

in settings/movement.py we have `self.movement = args.movement or MovementActions.AUTO_SPRINT` which means we will use auto sprint whether or not `-as` was ever specified

This means that vanilla movement behavior must be opt-in. I think that's probably better for the randomizer overall as it is a huge QOL (and the applications of forcing player to walk are probably very niche)

Reply:
(re-added on line 23)